docs/blog/2023-02-17-how-to-use-drand.md

```diff
@@ -141,8 +143,10 @@ const participants = ["alice", "bob", "carol", "dave", "edward", "fiona", "georg
 const chain = new HttpCachingChain("https://api.drand.sh") 
 const client = new HttpChainClient(chain)
 fetchBeaconByTime(client, Date.now()).then(beacon => {   
-	const winnerIndex = parseInt(Number("0x" + beacon.randomness.slice(0,2))) % participants.length // there are 7 participants, we take 1 byte of output
-	console.log(`the winner is ${participants[winnerIndex]}`) 
+  // there are 7 participants, we take 1 byte of output (!! bad !!)
+  const randomNumber = BigInt("0x" + beacon.randomness.slice(0, 2)) 
+  const winnerIndex = randomNumber % BigInt(participants.length)
+  console.log(`the winner is ${participants[winnerIndex]}`) 
```

Comment:
Is it legal to use a BigInt as an index? I'd be surprised of that works

Reply:
yup, it works fine! At least both in node and browser where I tested it

Reply:
Sweet! Always full of surprises this language